opscenter/example-server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
adding this to the doco will mean that there is a good chance we will end up with this cert used an many places, can you remove it from the docs?